submissions/sofiichuk/pop-up-task/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <title>pop-up task</title>
+    <meta charset="UTF-8">
+</head>
+<body>
+<nav class="navBar">
+    <ul>
+        <li class="navBar__item_txt"><a href="#">Mail</a></li>
+        <li class="navBar__item_txt"><a href="#">Images</a></li>
+        <li class="navBar__item_btn">
+            <input type="checkbox" id="input_1">
+            <label for="input_1">
+                <img src="../../../../popup_task/icons/dropdown_menu.png" alt="dropdown menu">
+            </label>
+            <nav class="dropdownMenu">
+                <ul>
+                    <li><a href="#"><img src="../../../../popup_task/icons/calendar.png" alt="calendar"/><span>Calendar</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/mail.png" alt="mail"/><span>Mail</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/news.png" alt="news"/><span>News</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/maps.png" alt="map"/><span>Map</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/files.png" alt="files"/><span>Files</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/contacts.png" alt="contacts"/><span>Contacts</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/translation.png" alt="translate"/><span>Translate</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/videos.png" alt="videos"/><span>Videos</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/search.png" alt="search"/><span>Search</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/collection.png" alt="collection"/><span>Collection</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/apps.png" alt="apps"/><span>Apps</span></a></li>
```

Comment:
Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

Reply:
I thought about it. And read articles. And came to conclusion that one word describing each menu item is enough – it’s brief & clear at the same time. So I left it as it had been.

Reply:
One of the points in the provided article is: for icons implemented with `img` tag `alt` attribute value must be an empty value, because Icon is a decorative image.

Reply:
Got it. Thanks. Removed alt values in cases of tautology.